### PR TITLE
Modification of release process for kustomize

### PR DESCRIPTION
**Pull request by jfilipcz**
That PR will change the target files we're setting versions on during the release.

### CODE REVIEW

**Comment by oybed** [comment]
Also need to update the following location:
https://github.com/rht-labs/lodestar-deployment/blob/master/bootstrap/kustomization.yaml#L125-L134

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -68,6 +68,46 @@ jobs:
...
+      SFrontVer: 'lodestar-frontend:  ${{ github.event.client_payload.slash_command.frontend }}'
+      SBackVer: 'lodestar-backend: ${{ github.event.client_payload.slash_command.backend }}'
+      SGitApiVer: 'lodestar-git-api: ${{ github.event.client_payload.slash_command.gitapi }}'
+      SStatusVer: 'lodestar-status: ${{ github.event.client_payload.slash_command.status }}'
+      SConfigVer: 'lodestar-config: '    #### To be filled in
+      SDispatchVer: 'resource-dispatcher: ${{ github.event.client_payload.slash_command.dispatcher }}'
+      SAgnosticVVer: 'agnosticv-operator: ${{ github.event.client_payload.slash_command.agnosticv }}'
+      SAnarchyVer: 'anarchy: ${{ github.event.client_payload.slash_command.anarchy }}'
+      SPoolBVer: 'poolboy: ${{ github.event.client_payload.slash_command.poolboy }}'
+      SLodeStarVer: 'lodestar: ${{ github.event.client_payload.slash_command.lodestar }}''
```
Comment (oybed) [inline .github/workflows/release.yml:109]: LodeStar version is obtained from the issue title - not by the slash_command - as seen here:
https://github.com/rht-labs/lodestar-deployment/blob/master/.github/workflows/release.yml#L134

**Review by oybed — CHANGES_REQUESTED** [review]
Looks like a great enhancement overall - good job.

A few inline comments - including a couple of nitpicks

File: bootstrap/patches/lodestar-git-api.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  namespace: lodestar-argo-cd
+spec:
+  ignoreDifferences:
+  - group: apps.openshift.io
+    jsonPointers:
+    - /spec/template/spec/containers/0/image
+    kind: DeploymentConfig
+  source:
+    helm:
+      values: 'imageTag: master'
```
[the remark this answers is not in the document]
Comment (jfilipcz) [reply bootstrap/patches/lodestar-git-api.yaml:14]: Unfortunately that's how yq outputs single line strings, I will check if we can change the default behavior

File: .github/workflows/release.yml (hunk 1 above)
Comment (jfilipcz) [reply .github/workflows/release.yml:109]: I reverted the change, now Lodestar version is populated with issue title

**Pull request merged**